src/client/components/Reservations/MealDetails.jsx

```diff
@@ -0,0 +1,42 @@
+import { useParams, Redirect, Link } from "react-router-dom";
+import React, { useContext } from "react";
+import { MealContext } from "../MealContext";
+import ReservationForm from "./ReservationForm";
+import ReviewForm from "../Reviews/ReviewFrom";
+
+function MealDetails() {
```

Comment:
This is conflicting with the Meal component. 
Are both needed? Could it be reorganized in some way? 🤔

Reply:
I did change this on my 3 homework delivery :)